src/Microsoft.Extensions.ProjectModel.MsBuild.Sources/MsBuildProjectContextBuilder.cs

```diff
@@ -37,7 +42,7 @@ public IProjectContext Build()
                     "msbuild",
                     _projectPath,
                     $"/t:EvaluateProjectInfoForCodeGeneration", 
-                    $"/p:TargetFramework={_targetFramework.GetShortFolderName()};OutputFile={tmpFile}"
+                    $"/p:OutputFile={tmpFile};CodeGenerationTargetLocation={_targetLocation}"
```

Comment:
It is pointing to the NuGet cache. 
https://github.com/aspnet/Scaffolding/pull/314/files#diff-ec1d0a509dd9dd0129c60bd0c0393efeR168

If you look at the test projects, I am deliberately taking a dependency on `Microsoft.VisualStudio.Web.CodeGeneration.Tools` so that the targets are included by restore.